src/components/inspectors/InspectorPanel.vue

```diff
@@ -91,10 +92,12 @@ export default {
     };
   },
   watch: {
-    highlightedNode() {
-      document.activeElement.blur();
-      this.prepareData();
-      this.prepareConfig();
+    highlightedNode(nVal, oVal) {
+      if (!isEqual(nVal, oVal)){
```

Comment:
add space before  {

Reply:
Why are you using `isEqual`
If the `nVal` and `oVal` are objects what happens if they are pointing to the same object? isEqual wil always return `true`.
Does it could happen? If that is the case the blur and prepare actions will not be executed in that case.

Reply:
I am using isEqual because the comparation is deep with this function,
The modeler fires two selection events twice, once when the mouse presses down and up,
This triggers calls to the inspector twice.
document.activeelement is related to the body and not to the modeler